### PR TITLE
Add livebook example

### PR DESCRIPTION
**Pull request by LauraBeatris**
Rebased against #37 

Allows developers to test this library by using Livebook. I've added the file pointing to the major release but locally I'm testing by pointing to my local build. 

Here's example of how it looks like initially, being able to instantiate a client: 

CleanShot 2023-10-21 at 22 05 54

It will later be added to the `README.md` file with a "Run on Livebook" button

### CODE REVIEW

File: config/test.ex [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import Config
+
+if workos_api_key =
+     System.get_env("WORKOS_API_KEY") and workos_client_id = System.get_env("WORKOS_CLIENT_ID") do
```
Comment (marktran) [inline config/test.ex:4]:
```suggestion
   System.get_env("WORKOS_API_KEY") and workos_client_id = System.get_env("WORKOS_CLIENT_ID") do
```

**Review by marktran — APPROVED** [review]

**Comment by LauraBeatris** [comment]
I made a bit of a mistake by running `git rebase origin/main` instead of running against `release/1.0.0`, will fix it now. This PR should only include the Livebook file.

File: lib/workos/passwordless/passwordless.ex [hunk 2]
```diff
@@ -36,7 +36,7 @@ defmodule WorkOS.Passwordless do
   def create_session(params, opts)
       when is_map_key(params, :email) or is_map_key(params, :connection) do
     query =
-      process_params(params, [:email, :connection, :redirect_uri, :state, :type], %{
+      process_params(params, [:email, :connection, :redirect_uri, :state, :type, :expires_in], %{
```
Comment (LauraBeatris) [inline lib/workos/passwordless/passwordless.ex:39]: Going to include this fix as well that was just merged in `main`

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+### Environment
+
+* Elixir version (elixir -v):
```
Comment (LauraBeatris) [inline ISSUE_TEMPLATE.md:3]: And will also include the `ISSUE_TEMPLATE` that was also merged in `main` 😅

File: workos_elixir.livemd [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# WorkOS + Elixir
+
+```elixir
+Mix.install([
+  {:workos, "~> 1.0.0"},
+  {:kino, "~> 0.9.4"}
+])
+```
+
```
Comment (LauraBeatris) [inline workos_elixir.livemd:9]: I will start adding more stuff here while refactoring the API modules

**Pull request merged**